ambry-utils/src/main/java/com.github.ambry.utils/Utils.java

```diff
@@ -43,6 +45,20 @@
  * A set of utility methods
  */
 public class Utils {
+
+  // epoch time beyond this date(time) will have issues
+  static String maxAllowableDateForEpochTime = "19-01-2038";
+  static String maxAllowableDateForEpochTimeFormat = "dd-MM-yyyy";
+  public static long maxEpochTimeInMs;
```

Comment:
Also why do we need the date and stuff ? I'd vote for just make this a constant since it is known beforehand